Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -340,11 +340,11 @@ keys read."))
   ((pointer)
    (cursor :initarg :text-cursor)))
 
-(defmethod handle-event :after
+(defmethod handle-event
     ((client standard-extended-input-stream) (event key-press-event))
   (stream-append-gesture client event))
 
-(defmethod handle-event :after
+(defmethod handle-event
     ((client standard-extended-input-stream) (event pointer-event))
   (stream-append-gesture client event))
```

Comment:
Doesn't this require subclasses to `(call-next-method)` where previously they did not have to?

Reply:
Only when they define their own handle-event method on the pointer-event or key-press-event. this does not occur, and for clients outside McCLIM that would be mixing two different abstractions.

Reply:
`clim-demo.drawing-tests::drawing-app-pane` and `drei:drei-gadget-pane` have such methods. Should those be changed to call the next method or are they problematic even before this change due to abstraction mixing?

Reply:
mixing abstractions is fine, but it requires additional care, that's what I've meant. A casual CLIM programmer doesn't have to do that. adding cal-next-method in them should be fine. I'll pick up from here tomorrow (afk).

Reply:
fixed